File: 프로그래머스/1/77484. 로또의 최고 순위와 최저 순위/로또의 최고 순위와 최저 순위.java
import java.util.*;
class Solution {
    public int[] solution(int[] lottos, int[] win_nums) {
        int[] answer = {0, 0};

        for(int l : lottos){
                if(l == 0) {
                    answer[0]++;
                    continue;
                }
            for(int w : win_nums){
                if(l == w){
                    answer[0]++;
                    answer[1]++;
                }
            }
        }
        
        answer[0] = Math.min(7 - answer[0], 6);
        answer[1] = Math.min(7 - answer[1], 6);
        
        return answer;
    }
}

File: 프로그래머스/1/77484. 로또의 최고 순위와 최저 순위/로또의 최고 순위와 최저 순위.py
def solution(lottos, win_nums):
    answer = [0, 0]
    rank = [6,6,5,4,3,2,1]
    max = 0
    min = 0
    
    for j in lottos:
        if j == 0:
            max += 1
        elif j in win_nums:
            min += 1

    answer = [rank[max + min], rank[min]]
    
    return answer